theme/js/components/version-selector-service.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const $ = require("./../jquery");
+
+class VersionSelectorService {
+  static _versions;
+
+  static _versionsPromise;
+
+  static getVersions = function () {
+    if (this._versionsPromise) {
+      return this._versionsPromise;
+    } else if (this._versions) {
+      return Promise.resolve(this._versions);
```

Comment:
`Promise is undefined` in IE11.

Maar waarom zou je een nieuwe promise teruggeven? Je kan toch prima de promise van `$.ajax()` bewaren en bij een volgend `getVersions()` uitgeven? Kun je ook de `static _versions` loslaten, die is dan ook niet nodig.

https://dso-toolkit.nl/_698-versions-selector-service/docs/versions.html